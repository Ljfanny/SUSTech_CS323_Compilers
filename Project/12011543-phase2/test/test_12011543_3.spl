int test_12011543_3()
{
  int abc;
  int a[1201][1543][1191][1212], b[1543][1191][1212], c[1191][1212], d[1212], e;
  int x[1212][1191][1543][1201], y[1212][1191][1543];
  a[0] = b;
  a[0][1201] = c;
  a[0][1201][1543] = d;
  a[0][1201][1543][1191] = e;
  e[1] = e;
  e = abc(e);
  return a[e][e][e][e];
}