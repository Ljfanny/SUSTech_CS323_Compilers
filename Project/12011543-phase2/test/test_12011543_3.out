Error type 10 at Line 10: indexing on non-array variable
Error type 5 at Line 10: unmatching types on both sides of assignment
Error type 11 at Line 11: invoking non-function variable: abc
Error type 5 at Line 11: unmatching types on both sides of assignment
